hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesCapacitySched.java

```diff
@@ -332,7 +336,7 @@ public void verifySubQueueXML(Element qElem, String q,
           for (int k = 0; k < qListInfos.getLength(); k++) {
             Element qElem3 = (Element) qListInfos.item(k);
             String qName3 = WebServicesTestUtils.getXmlString(qElem3, "queueName");
-            String q3 = q + "." + qName3;
+            String q3 = queue + "." + qName3;
```

Comment:
q3 will use only once for a function which needs a string, therefore I think this code part is not necessary to modify.

Reply:
Lets make a static utility function on QueuePath then, as the following:
```java
  public static String concatenatePath(String... parts) {
    return String.join(DOT, parts);
}
```
This will be useful for testing purposes as well.